WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -44,8 +44,15 @@ public async Task BuildPrivacySuggestionsAsync(TransactionInfo info, BuildTransa
 
 		var result = await _privacySuggestionsModel.BuildPrivacySuggestionsAsync(info, transaction, cancellationToken);
 
-		Warnings.AddRange(result.Warnings);
-		Suggestions.AddRange(result.Suggestions);
+		await foreach (var warning in result.GetAllWarningsAsync())
+		{
+			Warnings.Add(warning);
+		}
+
+		await foreach (var suggestion in result.GetAllSuggestionsAsync())
+		{
+			Suggestions.Add(suggestion);
+		}
```

Comment:
Rider suggests me this, does it make sense?

```suggestion
		await foreach (var warning in result.GetAllWarningsAsync().WithCancellation(cancellationToken))
		{
			Warnings.Add(warning);
		}

		await foreach (var suggestion in result.GetAllSuggestionsAsync().WithCancellation(cancellationToken))
		{
			Suggestions.Add(suggestion);
		}
```

Reply:
I don't think we need to pass cancellationToken to these, because we're already handling that in the "source" async enumerables (remember the `PrivacySuggestionsResult` is just a wrapper for that)